.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -0,0 +1,42 @@
+# TODO(FMP-225):
+#
+#   - create a ticket for deploying ui (demo.fmprotocol.com)
```

Comment:
Since pull requests are working, I should split this linear ticket into two, and use the new ticket to track the work to get the other workflow file working.